contrib/jmx-metrics/src/main/java/io/opentelemetry/contrib/jmxmetrics/JmxConfig.java

```diff
@@ -69,6 +71,10 @@
     groovyScript = properties.getProperty(GROOVY_SCRIPT);
     targetSystem = properties.getProperty(TARGET_SYSTEM, "").toLowerCase().trim();
 
+    List<String> targets =
+        Arrays.asList(isBlank(targetSystem) ? new String[0] : targetSystem.split(","));
+    targetSystems = new LinkedHashSet<>(targets).toArray(new String[0]);
```

Comment:
Is this to remove duplicates?

Reply:
Right, it's my understanding that a `LinkedHashSet` preserves insertion order as well.